tests/decorator_test.py

```diff
@@ -18,46 +18,44 @@ def func(self) -> None:
         def no_docstring(self) -> None:
             pass
 
-    class CorrectImplementation(BaseClass):
+    class CorrectImplementation(BaseClass):  # pyright: ignore
```

Comment:
what's pyright's complaint here? we could maybe delete this test alltogether, because we deprecated `@implements`.

Reply:
that the class is unused (I tried to specify the error to be ignore but had no luck getting it to work)

Reply:
Let's just leave the test there for now; it's not doing any harm